conformance/lang/expressions/range-expr/range_expr_exclusive.balt

```diff
@@ -0,0 +1,5001 @@
+Test-Case: output
+Description: Test range expression (exclusive) when values of both expressions are positive integers.
+Labels: range-expr, iterable-exclusive, int, DecimalNumber, additive-expr
+
+function init() {
+    int startIndex = 0;
+    int endIndex = 6;
+    int count = 0;
+    foreach int i in startIndex ..< endIndex {
```

Comment:
For each of these tests shouldn't we actually print and make use of `i`?  We aren't quite testing the values returned by the range expression if not.